src/index.html

```diff
@@ -4,15 +4,24 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,700&display=swap">
+    <link rel="stylesheet" href="css/style.css">
     <title>freeCodeCamp Manila</title>
   </head>
   <body>
-    <h1>Website is under construction!</h1>
-    <h2>freeCodeCamp Manila</h2>
-    <p>
-      Want to contribute? Check our
-      <a href="https://github.com/fccManila/fccmanila.github.io">Github</a>
-      repo.
-    </p>
+    <header class="header">
+      <div class="container">
+        <a href="#" class="fcc-logo">
+          <img class="fcc-logo_img" src="images/freeCodeCamp-logo.png" alt="FreeCodeCamp Manila">
+          freeCodeCamp Manila
+        </a>
+        <nav class="header-nav">
+          <a class="header-nav_link" href="#">Blog</a>
+          <a class="header-nav_link" href="#">Leaderboard</a>
+          <a class="header-nav_link fcc-community_btn" href="#">Join the Community</a>
```

Comment:
Here's the link of the Join the Community button `https://facebook.com/groups/free.code.camp.manila/`

Reply:
Do I need to target it to a new Tab?

Reply:
Yes, please 😄

Reply:
Also please add `rel="noopener noreferrer"`, [here's why](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/).

Reply:
Ok.